.github/ISSUE_TEMPLATE/new-source-suggestion.md

```diff
@@ -1,17 +1,19 @@
 ---
-name: New Source Suggestion for CC Search
-about: Tell us about a website or platform with CC-licensed content
-title: "[Source Suggestion] Insert source name here"
-labels: awaiting triage, ticket work required, providers
-assignees: ''
-
+name: New source suggestion
+about: Suggest a site/platform with CC-licensed content to include in CC Search
+labels: "🛠 goal: addition, 🚦 status: awaiting triage, 💻 aspect: code, 🟩 priority: low"
```

Comment:
This isn't really a coding issue.  It's a research task.  Is there a different aspect that would be more relevant?

Reply:
I created a new repo-level label `aspect: research` and updated the templates to use that.

Reply:
Resolved in 23a73e6.